### ISSUE TITLE
[Feature Request] v-combobox prop to clear input on select of a matched item because the input is no longer useful

### ISSUE THREAD

**Issue by JessePerry** [vuetifyjs/vuetify#11804]
### Problem to solve
When you type the start of a search in a combobox, eg: "Je" and this matches with an entry, and shows a dropdown of checkboxes by default with options: eg: "Jesse".
You click that entry and it is added to the input part, which is good, but you are still left with your partial input which is now not useful, this is the kind of thing you would see:
Jesse, Je|

I can't customise the v-combobox to clear the input because the v-combobox does not expose the `<input>` part of the component with any scope or prop. So we can’t inspect it (or clear it) before it’s an item in the list.
Only after the user presses Space or Enter, and the chip item is created, then it’s accessible to any script or function. 
But before that when it’s text entry, it’s not possible to edit it or clear it back to nothing. 

I tried making a scoped item for the selectable checkboxes, which worked, and putting a clicked event on them, and that’s when I could clear the rest of the incomplete input, except that it’s not reachable.

### Proposed solution
The input should be cleared after a match is selected because the input is not useful to the user anymore.
If this is a toggle option then a prop on v-checkbox would be great like `clear-on-select`

**Comment by palmerye** [comment]
@JessePerry This problem has happened to me in the past.
I solved the problem like this.

``` javascript
<v-combobox ref="myComboBox" v-model="selectItem"></v-combobox>

...

watch: {
    selectItem () {
         this.$refs.myComboBox.internalSearch = ''
    }
}
```

**Comment by JessePerry** [comment]
Thanks so much @palmerye! That's an excellent workaround.

How did you find out that internalSearch was a property on the v-combobox? I looked everywhere for docs about this but couldn't find any.

**Comment by palmerye** [comment]
@JessePerry , Vue-Devtools may help you to find some property that can not be got in the docs.

![](https://user-images.githubusercontent.com/16277812/87318683-40ea1580-c55b-11ea-80e9-765c84fdd155.png)

**Comment by KaelWD** [comment]
This is exposed as the `search-input.sync` two-way prop as well.

**Comment by KaelWD** [comment]
Also duplicate of #8842

**Comment by agusmaulanayusufnoor** [comment]
this.$refs.myComboBox.reset();

just work for me
